src/libraries/System.Private.CoreLib/src/System/Numerics/INumber.cs

```diff
@@ -154,7 +154,7 @@ static virtual TSelf MinNumber(TSelf x, TSelf y)
 
         /// <summary>Computes the sign of a value.</summary>
         /// <param name="value">The value whose sign is to be computed.</param>
-        /// <returns>A positive value if <paramref name="value" /> is positive, <see cref="INumberBase{TSelf}.Zero" /> if <paramref name="value" /> is zero, and a negative value if <paramref name="value" /> is negative.</returns>
+        /// <returns>A positive value if <paramref name="value" /> is positive, <c>0</c> if <paramref name="value" /> is zero, and a negative value if <paramref name="value" /> is negative.</returns>
```

Comment:
This is intentional as it is meant to return the `INumberBase{TSelf}.Zero` property. This may or may not be the same as the numeric literal `0`

Reply:
The return type of this method is `int`.

Reply:
Ah, my bad. I forgot this is one of the few cases that differs like that.